conjur/constant.py

```diff
@@ -0,0 +1,5 @@
+"""
+File to hold program constants
+"""
+
+ARG_ERROR_FORMAT = 'unrecognized arguments: %s'
```

Comment:
I don't think that having a single constant that's not reused anywhere in a separate module give us any value. I don't see why this couldn't be a constant within the wrapper itself.

Reply:
💯 removing